src/render/update.py

```
@@ -25,7 +29,7 @@ def get_sequence():
 
 
 # Ignore raw data for caching, too ressource intensive
-hash_funcs = {pd.DataFrame : lambda x : 1}
+hash_funcs = {pl.LazyFrame : lambda x : 1}
```

Ruff (0.12.2)

Unused lambda argument: `x`

(ARG005)